airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json

```diff
@@ -45,6 +45,27 @@
         "description": "Whether or not to normalize the data in the destination. See <a href=\"https://docs.airbyte.io/architecture/basic-normalization\">basic normalization</a> for more details.",
         "title": "Basic Normalization",
         "examples": [true, false]
+      },
+      "s3_bucket_name" : {
```

Comment:
I spent 2 hours or so running into some frontend weirdness with spec.json rendering. I decided to cut my losses and go with this for now. What we lose is 1 additional authentication method + some JSON niceties. I feel more valuable to release what we have not and bolt the rest of this on later. I'm also ready to stop working on Redshift :P  Does that sound sane?

Any bad input is caught by the https://github.com/airbytehq/airbyte/pull/2606/files#diff-719915fe360d60f49387522687fbde9a7ae807bc837c319edb242b977359e417R202 for the time being. Will write a ticket describing what I was trying to do and what did/did not work.